### PR TITLE
lib/vfscore: Fix spelling mistake in `Config.uk`

### PR DESCRIPTION
**Pull request by b1shtream**
### Prerequisite checklist

 - [x] Read the [contribution guidelines](https://unikraft.org/docs/contributing/) regarding submitting new changes to the project;
 - [x] Tested your changes against relevant architectures and platforms;
 - [x] Ran the [`checkpatch.uk`](https://github.com/unikraft/unikraft/blob/staging/support/scripts/checkpatch.uk) on your commit series before opening this PR;
 - [x] Updated relevant documentation.

### Base target

 - Architecture(s): [e.g. `x86_64` or N/A]
 - Platform(s): [e.g. `kvm`, `xen` or N/A]
 - Application(s): [e.g. `app-python3` or N/A]

### Description of changes

Fixed typo in line 32
`bool "Automatically mount a root filesysytem (/)"` ---> `bool "Automatically mount a root filesystem (/)"`

### CODE REVIEW

File: lib/vfscore/Config.uk [hunk 1]
```diff
@@ -29,7 +29,7 @@ config LIBVFSCORE_NONLARGEFILE
 		the 64-bit version of the system calls are registered.
 
 config LIBVFSCORE_AUTOMOUNT_ROOTFS
-bool "Automatically mount a root filesysytem (/)"
+bool "Automatically mount a root filesystem (/)"
```
Comment (skuenzer) [inline lib/vfscore/Config.uk:32]: Thanks for this fix :+1:. A few minor comments:
- Can you update your commit title to use `lib/vfscore: Fix spelling mistake in Config.uk`, too? I think this is better readable with `git log`.
- Can you address the other issues that checkpatch complains about in your commit message (mostly line length)?
- We use the `Signed-off-by` tag as an act of signing your contribution that it complies with the [Certificate of Origin](https://unikraft.org/docs/contributing/unikraft#developers-certificate-of-origin). Would you be able to use a proper mail address of yours which is not a "noreply" address?

Thanks!

**Review by skuenzer — CHANGES_REQUESTED** [review]

File: lib/vfscore/Config.uk (hunk 1 above)
Comment (b1shtream) [reply lib/vfscore/Config.uk:32]: Hello @skuenzer 
I have done the following changes, kindly review them once again. Thank you!

**Review by John-Ted — APPROVED** [review]
Looks good to me, thanks!
Reviewed-by: Ioan-Teodor Teugea ioan_teodor.teugea@stud.acs.upb.ro

**Review by razvand — APPROVED** [review]
Approved-by: Razvan Deaconescu <razvand@unikraft.io>

**Pull request merged**